<launch>
  <arg name="use_robot_name"         default="elpistar"/>
  <arg name="device_name"            default="/dev/ttyUSB0"/>
  <arg name="baud_rate"              default="1000000"/>
  <arg name="protocol_version"       default="1.0"/>
  <arg name="joint_info"             default="$(find elpistar_dynamixel)/config/front_standup.yaml"/>
  <arg name="joint_controller"       default="position_mode"/>

  <node pkg="elpistar_dynamixel" type="elpistar_dynamixel_ctrl" name="elpistar_dynamixel_controller" required="true" output="screen">
    <param name="robot_name"           value="$(arg use_robot_name)"/>
    <param name="device_name"          value="$(arg device_name)"/>
    <param name="baud_rate"            value="$(arg baud_rate)"/>
    <param name="protocol_version"     value="$(arg protocol_version)"/>
    <param name="dynamixel_info"       value="$(arg joint_info)"/>
    <param name="joint_controller"     value="$(arg joint_controller)"/>
    
  </node>
</launch>
